operator/gitops/argocd/pipeline-service/metrics-exporter/kustomization.yaml

```diff
@@ -10,7 +10,7 @@ resources:
 images:
   - name: quay.io/redhat-appstudio/pipeline-service-exporter
     newName: quay.io/redhat-appstudio/pipeline-service-exporter
-    newTag: 329e1a6617af1740ad133a8338cb4c328dcc7f24
+    newTag: 089f2e576d31759eca98f78a591deda2cb5f37d4
```

Comment:
tag is up at https://quay.io/repository/redhat-appstudio/pipeline-service-exporter?tab=tags